systemtest/src/test/resources/io/strimzi/systemtest/ConnectClusterIT.testForUpdateValuesInConnectCM.yaml

```diff
@@ -4,5 +4,11 @@ metadata:
   name: my-cluster
 spec:
   replicas: 1
+  readinessProbe:
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  livenessProbe:
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
```

Comment:
Why did this need to be changed?

Reply:
The problem here that we will have `NPE` in line https://github.com/strimzi/strimzi-kafka-operator/blob/e3e71335d4628319bee9fdd24d75136f428645da/systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java#L172 if we don't have `livenessProbe` and `readinessProbe` in .yaml file for deploying Kakfa Connect.

Reply:
@tombentley Can it be engineering bug?